### DISCUSSION TITLE
firebase cpp sdk or separate platform specific native sdk?

### DISCUSSION THREAD

**Discussion by meehjeevan** [axmolengine/axmol#1997] [Q&A]
So, if I have to use admob and other firebase services and I need to target for android and ios then what would be the best option? Coding in c++ using cpp sdk within main c++ source files or coding in java for android and in objective-c for ios separately?

**Comment by paulocoutinhox** [comment]
Hi,

I use the SDK from each platform using my project on my games:
https://github.com/paulocoutinhox/axmol-native-helper

That has a sample working for each platform.

You can use iOS SDK and Android SDK without problems.

Maybe it help you.
